definitions/procedures/backup/config_files.rb

```diff
@@ -11,16 +11,18 @@ class ConfigFiles < ForemanMaintain::Procedure
       param :backup_dir, 'Directory where to backup to', :required => true
       param :proxy_features, 'List of proxy features to backup (default: all)',
             :array => true, :default => ['all']
+      param :strategy, 'Backup strategy'
```

Comment:
This could be renamed to :ignore_changed_files or similar with  `:flag => true, :default => false` options